src/input/reader_to_storage.cc

```diff
@@ -632,7 +633,7 @@ StorageBase * ReaderToStorage::make_storage_from_default(const string &dflt_str,
 
```

Comment:
Either I attached the comment to the wrong code or was unconscious, this code is pretty clear.